content/en/announcements/otel-localized.md

```diff
@@ -0,0 +1,13 @@
+---
+title:
+  中文, 日本語, Português — OpenTelemetry documentation is now available in
+  three more languages
+date: 2024-08-21
+expiryDate: 2024-09-30
+weight: -1 # top
+cSpell:ignore: Português
+---
+
+<i class="fas fa-language"></i> 中文, 日本語, Português — OpenTelemetry
+documentation is now available in three more languages!
+[Learn more about our efforts, to make OpenTelemetry available in your language!](/blog/2024/docs-localized)
```

Comment:
Actually, I find this too long for a banner. How about, more concisely (details will be in the blog):

```suggestion
<i class="fas fa-language"></i> OpenTelemetry website goes multilingual! 
[Learn more](/blog/2024/docs-localized)
```